app/javascript/dashboard/components/ui/Banner.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="banner" :class="bannerClasses">
+    <span>
+      {{ bannerMessage }}
+      <a
+        v-if="hrefLink"
+        :href="hrefLink"
+        rel="noopener noreferrer nofollow"
+        target="_blank"
+      >
+        {{ hrefLinkText }}
+      </a>
+    </span>
+    <woot-button
+      v-if="hasActionButton"
+      size="small"
+      variant="link"
+      icon="arrow-right"
+      color-scheme="primary"
+      class-names="banner-action__button"
+      @click="onClick"
+    >
+      {{ actionButtonLabel }}
+    </woot-button>
+    <woot-button
+      v-if="hasCloseButton"
+      size="small"
+      variant="link"
+      color-scheme="warning"
+      icon="dismiss-circle"
+      class-names="banner-action__button"
+      @click="onClickClose"
+    >
+    </woot-button>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    bannerMessage: {
+      type: String,
+      default: '',
+    },
+    hrefLink: {
+      type: String,
+      default: '',
+    },
+    hrefLinkText: {
+      type: String,
+      default: '',
+    },
+    hasActionButton: {
+      type: Boolean,
+      default: false,
+    },
+    actionButtonLabel: {
+      type: String,
+      default: '',
+    },
+    bgColorScheme: {
```

Comment:
```suggestion
    ColorScheme: {
```

Color-scheme is enough.

Reply:
Fixes by https://github.com/chatwoot/chatwoot/pull/3867/commits/dbbd0bc22944ac95dd4e78e79d8cc5893f42e303